spring-rabbit-stream/src/main/java/org/springframework/rabbit/stream/listener/StreamListenerContainer.java

```diff
@@ -117,6 +118,13 @@ public StreamListenerContainer(Environment environment, @Nullable Codec codec) {
 						: new DefaultStreamMessageConverter();
 	}
 
+	/**
+	 * Get a name of stream this listener listens to.
```

Comment:
```suggestion
	 * Get a stream name this listener is subscribed to.
	 * @return the stream name this listener is subscribed to.
	 * @since 3.2.3
```